shunit2

```diff
@@ -867,31 +867,31 @@ _shunit_generateReport() {
   _shunit_ok_=${SHUNIT_TRUE}
 
   # If no exit code was provided one, determine an appropriate one.
-  [ ${__shunit_testsFailed} -gt 0 \
+  \[ ${__shunit_testsFailed} -gt 0 \
       -o ${__shunit_testSuccess} -eq ${SHUNIT_FALSE} ] \
           && _shunit_ok_=${SHUNIT_FALSE}
 
   echo
-  if [ ${__shunit_testsTotal} -eq 1 ]; then
-    ${__SHUNIT_CMD_ECHO_ESC} "Ran ${__shunit_ansi_cyan}${__shunit_testsTotal}${__shunit_ansi_none} test."
+  if \[ ${__shunit_testsTotal} -eq 1 ]; then
+    echo "Ran ${__shunit_testsTotal} test."
```

Comment:
@kward I think I made a mistake here while I was resolving all those merge conflicts during the rebase - however, it looks like you have done quite a bit of refactoring since this was merged and I think you already fixed it? Anyhow, just a heads up in case you didn't notice.